### ISSUE TITLE
Entity view visual bug

### ISSUE THREAD

**Issue by Zasa-san** [huridocs/uwazidocs#5397]
**Describe the bug**
Entity view breaks when a relationship property points to an entity with a long name.

**To Reproduce**
This can be seen on demo instance: https://demo.uwazi.io/en/entity/ios49y7turh

**Screenshots**
![image](https://user-images.githubusercontent.com/71732018/207128950-61ea8274-734e-4a4e-8582-0c711942e543.png)

**Expected behavior**
It should look similar to the following screenshot, having some max width for every column.
![image](https://user-images.githubusercontent.com/71732018/207129009-ba53172e-2a54-4906-9c4b-aea58ba3d31c.png)

**Device (please select all that apply)**
Desktop

**Browser**
Firefox

**Comment by RafaPolit** [comment]
I don't think this is a bug.  The "default" behavior is that it simply "shows as many columns as possible" depending on the content.  The content is unbreakable and spans for the entire width, so, if it's really long, you get a single column.

Still, this behavior can be (and is usually) easily adjustable with the Custom CSS for each collection.

The reason we introduced the no-wrapping white spaces was that, if there were multiple items and not being separated by comma (your second screenshot is the perfect example!), it was really hard to determine which line was the same link and where the new link started.  So, each link gets it's own line with a non-breaking rule.

So, imagine if the Firmantes section on your second screenshot had a space breaking rule? It would read something like:
```
Cecilia Medina
Quiroga
Diego-Garcia-Sayan
Leonarde A.
Franco
Manuel E.
Ventura Robles
...
```

See where I'm going?  You completely loose track of what the link name really is.

Makes sense? If you have an alternative, let's implement that, but please consider the above so that we don't re-introduce the problem that created this condition in the first place.

**Comment by Zasa-san** [comment]
Makes sense @RafaPolit. I'll remove the bug label.

If the line height was less than the padding/margin between items it would make breaks distinguishable from the next item in the list. We could implement that, or leave it as is and allow each user to decide how to handle these cases.
